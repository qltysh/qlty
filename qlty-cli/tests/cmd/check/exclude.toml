bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--verbose"]
status.code = 1
