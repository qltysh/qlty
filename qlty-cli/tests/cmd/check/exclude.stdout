
 ISSUES: 2 

:0:0
    0:0  high    exists failed  exists:fail
    0:0  high    exists failed  exists:fail

 JOBS: 2 

Plugin  Result   Targets   Time   Debug File
exists  Success  1 target  [..]s  .qlty/out/invoke-[..].yaml
exists  Success  1 target  [..]s  .qlty/out/invoke-[..].yaml

Checked 2 files
